caluma/caluma_analytics/simple_table.py

```diff
@@ -740,6 +754,116 @@ def root_fields(self):
         return {**direct_fields, **form_fields, **workitem_fields}
 
 
+class WorkItemsStartingObject(BaseStartingObject):
+    identifier = "work_items"
+    label = "Work Items"
+
+    def get_query(self):
+        return self.visibility_source.workitems()
```

Comment:
Can we rename this to `.work_items()` as well to be consistent?

Reply:
Done